src/index.html

```diff
@@ -5,10 +5,80 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
+
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div>
+        <a href="#home" class="nav__link">
+          <img
+            src="images/logo.png"
+            alt="Logo"
+            class="logo"
+          >
+        </a>
+      </div>
+
+      <nav>
+        <ul class="nav__list">
+          <li class="list__item">
+            <a href="#apple" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="list__item">
```

Comment:
add an empty line between siblings
```suggestion
          </li>
          
          <li class="list__item">
```